contactaccessrights/CRM/Contactaccessrights/Component/ContactActionsMenu/UserACLGroupsListItem.php

```diff
@@ -0,0 +1,7 @@
+<?php
+
+use CRM_Contactaccessrights_Component_ContactActionsMenu_GenericListItem as GenericListItem;
+
+class CRM_Contactaccessrights_Component_ContactActionsMenu_UserACLGroupsListItem extends GenericListItem {
+
+}
```

Comment:
We can but I thought of the following advantages:
- It will be easy to check the items in tests
- Also if any of the items need to be displayed differently later, it would be easier.  e.g its possible that the UserLocations item should be linked to their locations page and the User regions items be linked to the region page or the label for one is to be red and the other blue. 
The way it is, it would be easy to just add the render function for the ListItem that needs to be different in the future and nothing needs to be changed in the Action group helper class and tests.

Reply:
There's a difference in changing the class structure to make it easier to test them (this usually reduces coupling) and creating an empty class that isn't  used for anything but the test. In the future, if it really happens that we need a different type of list, we can just create a separate class for that.